plugins/modules/aci_l3out_logical_interface_profile_eigrp_profile.py

```diff
@@ -0,0 +1,319 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2024, Jason Juenger (@jasonjuenger) <jasonjuenger@gmail.com>
+# Copyright: (c) 2023, Gaspard Micol (@gmicol) <gmicol@cisco.com>
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {"metadata_version": "1.1", "status": ["preview"], "supported_by": "community"}
+
+DOCUMENTATION = r"""
+---
+module: aci_l3out_logical_interface_profile_eigrp_profile
```

Comment:
Should we keep the same naming as the ospf equivalent `aci_l3out_logical_interface_profile_ospf_policy`?

```suggestion
module: aci_l3out_logical_interface_profile_eigrp_policy
```

Reply:
I waffled back and forth on this one. In ACI it is named a profile rather than a policy, but, as you called out, the equivalent module for OSPF is named as a policy. I'm fine with either, I leave it up to you which name you prefer to go with.

Reply:
Well in ACI it is still called a policy only for this class it sits in a profile (eigrpIfP) which is a container like class that does not any identifiable attributes. So when you configure EIGRP you would in my view want to create the relationship towards the policy. So I would keep it consistent with the OSPF module naming.